### PR TITLE
Add Service Bus queues

### PR DESCRIPTION
**Pull request by paulsimonandrews**
https://eaflood.atlassian.net/browse/SF-146

### CODE REVIEW

File: app/config/messaging.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  host: process.env.MESSAGE_QUEUE_HOST,
+  password: process.env.MESSAGE_QUEUE_PASSWORD,
+  username: process.env.MESSAGE_QUEUE_USER
+}
+
+module.exports = {
+  updateEligibilityQueue: {
+    address: process.env.UPDATE_ELIGIBILITY_QUEUE_ADDRESS,
+    type: 'queue',
+    usePodIdentity: process.env.NODE_ENV === 'production',
```
Comment (st3v3nhunt) [inline app/config/messaging.js]: Given these are both queues atm, is it worth including `type` and `usePodIdentity` in the `connectionDetails` object and possible changing the name of it to shared config or something less specific to accommodate the extra properties?

File: package.json [hunk 2]
```diff
@@ -22,13 +22,16 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
+    "@azure/ms-rest-nodeauth": "^3.0.6",
+    "@azure/service-bus": "^1.1.10",
```
Comment (st3v3nhunt) [inline package.json]: Did you know these two deps needed to be installed along with `ffc-messaging`? I can't see anything in the `README`.

**Review by st3v3nhunt — COMMENTED** [review]
Looks good, just a couple of questions.

Regarding `update-agreement.js` and `update-eligibility.js` they are very similar. I know they aren't being used just yet but I was wondering if the data was 'typed' (e.g. a property added to the data to distinguish it) it would be possible to have a single `update-thing.js` module.

File: package.json (hunk 2 above)
Comment (paulsimonandrews) [reply package.json]: They are installed by `ffc-messaging` so will remove from the deps

File: app/config/messaging.js (hunk 1 above)
Comment (paulsimonandrews) [reply app/config/messaging.js]: Good point, will make it so.

**Review by st3v3nhunt — APPROVED** [review]
LGTM :+1:

**Pull request merged**